### PR TITLE
New method getEnrichedFacilities

### PR DESCRIPTION
**Pull request by cuadradek**
- New class EnrichedFacility was created, it represents facility with its hosts
and destinations.
- New method getEnrichedFacilities was created and tested.

### CODE REVIEW

File: perun-core/src/main/java/cz/metacentrum/perun/core/api/FacilitiesManager.java [hunk 1]
```diff
@@ -114,6 +114,15 @@
 	 */
 	List<Facility> getFacilities(PerunSession perunSession) throws PrivilegeException;
 
+	/**
+	 * Gets all enriched facilities.
```
Comment (stavamichal) [inline perun-core/src/main/java/cz/metacentrum/perun/core/api/FacilitiesManager.java:118]: It should be more like "Gets all enriched facilities user has access rights to." Similar to getFacilities javadoc.

File: perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/FacilitiesManagerMethod.java [hunk 2]
```diff
@@ -161,6 +161,19 @@ public Integer call(ApiCaller ac, Deserializer parms) throws PerunException {
 		}
 	},
 
+	/*#
+	 * Gets all enriched facilities.
```
Comment (stavamichal) [inline perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/FacilitiesManagerMethod.java:165]: The same change as for Entry.

**Review by stavamichal — CHANGES_REQUESTED** [review]

File: perun-core/src/main/java/cz/metacentrum/perun/core/api/FacilitiesManager.java (hunk 1 above)
Comment (cuadradek) [reply perun-core/src/main/java/cz/metacentrum/perun/core/api/FacilitiesManager.java]: Fixed.

File: perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/FacilitiesManagerMethod.java (hunk 2 above)
Comment (cuadradek) [reply perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/FacilitiesManagerMethod.java]: Fixed.

**Review by stavamichal — APPROVED** [review]

**Review by mvocu — APPROVED** [review]
Looks good.

Personally I do not like the concept of calling these classes, that serve only as messages in RPC, "Enriched", and mixing them with other classes that actually model something, but I understand this is a custom here.

**Pull request merged**